package.json

```diff
@@ -43,12 +43,12 @@
     "babel-plugin-transform-charcodes": "^0.2.0",
     "chalk": "^2.4.2",
     "charcodes": "^0.2.0",
-    "eslint": "^7.5.0",
-    "eslint-import-resolver-node": "^0.3.3",
-    "eslint-plugin-flowtype": "^4.6.0",
-    "eslint-plugin-import": "^2.22.0",
-    "eslint-plugin-jest": "^23.8.2",
-    "eslint-plugin-prettier": "^3.1.2",
+    "eslint": "^7.27.0",
+    "eslint-import-resolver-node": "^0.3.4",
+    "eslint-plugin-flowtype": "^5.7.2",
+    "eslint-plugin-import": "^2.23.4",
+    "eslint-plugin-jest": "^24.3.6",
+    "eslint-plugin-prettier": "^3.4.0",
```

Comment:
This is the root package, it is fine to bump major versions since we build on latest node.